dockstore-webservice/src/main/java/io/dockstore/webservice/core/Image.java

```diff
@@ -69,6 +69,10 @@
     @ApiModelProperty(value = "Registry the image belongs to", position = 5)
     private Registry imageRegistry;
 
+    @Column()
```

Comment:
Did not put a constraint. Other workflow/tool versions could contain the same docker image. Did not think it was worth guaranteeing uniqueness per entry version. I think I'd have to write a trigger since I used an extra table to map images to their versions (although maybe that should change..)

Reply:
Fair enough